exercises/accumulate/accumulate.py

```diff
@@ -1,2 +1,2 @@
-def accumulate():
+def accumulate(array, function):
```

Comment:
> As an improvement, I would name params with meaningful names rather than generic ones. But it's up to you to decide

I wanted to use names from example.py, but as @behrtam said in issue #509 choosing good name is important thing too. So I chose generic ones. But that is just detail which depends on exercise creator.